recipes/PySP2/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pysp2" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pysp2-{{ version }}.tar.gz
+  sha256: 234400bc9205bbe47659b407062a8885b309d19fa93ce2651b821af67d7cdf71
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - matplotlib-base
+    - numpy
+    - pandas
+    - scipy
+    - xarray
+    - xlrd
+    - act-atmos
+
+test:
+  imports:
+    - pysp2
+  commands:
+    - pytest 
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests/*.py
+
+about:
+  home: https://pypi.org/project/pysp2/
+  summary: Py-SP2
```

Sorry, I just noticed the summary was a bit...undescriptive. How about (from PyPI/GitHub):

```suggestion
  summary: A package for analyzing Single Particle Soot Photometer (SP2) data
  description: |
      This is a python package for processing and visualizing SP2 data. It is based off of the IGOR code developed by 
      Droplet Measurement Technologies. PySP2 currently supports processing all raw SP2 data (.sp2b, .hk, .ini) files 
      into mass and number size distributions. It can plot individual waveforms as well as particle size distributions. 
      Its file format is based off of the standard provided by the [Atmospheric Community Toolkit](https://arm-
      doe.github.io/ACT) which is based around [xarray](https://xarray.pydata.org/). PySP2 is currently used by the 
      Department of Energy Atmospheric Radiation Measurment (ARM) Facility in order to process SP2 observations 
      during field deployments such as TRACER and SAIL.
```